boards/ek-lm4f120xl/include/periph_conf.h

```diff
@@ -0,0 +1,385 @@
+
+/* Timer 1 configuration */
+#define TIMER_1_CHANNELS    2
+#define TIMER_1_PRESCALER   (39U)
+#define TIMER_1_MAX_VALUE   (0xffffffff)
+#define TIMER_1_ISR         isr_wtimer1
+#define TIMER_1_IRQ_CHAN    Timer1A_IRQn
+/** @} */
+
+/** @} */
+
+/**
+ * @name UART configuration
+ * @{
+ */
+#define UART_NUMOF          (1U)
+#define UART_0_EN           1
+#define UART_1_EN           0
+#define UART_IRQ_PRIO       1
+#define UART_CLK            ROM_SysCtlClockGet()  /*RT clock runs with 40MHz */
+/* UART 0 device configuration */
+#define UART_0_DEV          UART0_BASE
+#define UART_0_CLK          (40000000)          /* UART clock runs with 42MHz (F_CPU / 4) */
+#define UART_0_IRQ_CHAN     UART0_IRQn
+#define UART_0_ISR          isr_usart0
+/* UART 0 pin configuration */
+#define UART_0_PORT         GPIOA
+#define UART_0_TX_PIN       UART_PA1_U0TX
+#define UART_0_RX_PIN       UART_PA0_U0RX
+#define UART_0_AF           7
+
+
+
+/* UART 1 device configuration */
+#define UART_1_DEV          UART1_BASE
+#define UART_1_CLK          (40000000)          /* UART clock runs with 40MHz (F_CPU / 4) */
+#define UART_1_IRQ_CHAN     UART1_IRQn
+#define UART_1_ISR          isr_usart1
+/* UART 1 pin configuration */
+#define UART_1_PORT         GPIOD
+#define UART_1_TX_PIN       8
+#define UART_1_RX_PIN       9
+#define UART_1_AF           7
+/** @} */
+
+/**
+ * @name ADC configuration
+ * @{
+ */
+#define ADC_NUMOF           (2U)
+#define ADC_0_EN            1
+#define ADC_1_EN            1
+#define ADC_MAX_CHANNELS    2
+/** @} */
+/* ADC 0 configuration */
+#define ADC_0_DEV           ADC0_BASE
+#define ADC_0_CHANNELS      2
+#define ADC_0_PORT          GPIO_PORTE_BASE
+/* ADC 0 channel 0 pin config */
+#define ADC_0_CH0           1
+#define ADC_0_CH0_PIN       GPIO_PIN_3
+/* ADC 0 channel 1 pin config */
+#define ADC_0_CH1           4
+#define ADC_0_CH1_PIN       4
+
+/* ADC 1 configuration */
+#define ADC_1_DEV           ADC1_BASE
+#define ADC_1_CHANNELS      2
+
+#define ADC_1_PORT          GPIO_PORTE_BASE
+#define ADC_1_PORT_CLKEN()  (RCC->AHB1ENR |= RCC_AHB1ENR_GPIOCEN)
+/* ADC 1 channel 0 pin config */
+#define ADC_1_CH0           11
+#define ADC_1_CH0_PIN       GPIO_PIN_2
+/* ADC 1 channel 1 pin config */
+#define ADC_1_CH1           12
+#define ADC_1_CH1_PIN       2
+/** @} */
+
+
+
+/**
+ * @name DAC configuration
+ * @{
+ */
+#define DAC_NUMOF          (1U)
+#define DAC_0_EN           1
+#define DAC_MAX_CHANNELS   2
+
+/* DAC 0 configuration */
+#define DAC_0_DEV            DAC
+#define DAC_0_CHANNELS       2
+#define DAC_0_CLKEN()        (RCC->APB1ENR |=  (RCC_APB1ENR_DACEN))
+#define DAC_0_CLKDIS()       (RCC->APB1ENR &= ~(RCC_APB1ENR_DACEN))
+#define DAC_0_PORT           GPIOA
+#define DAC_0_PORT_CLKEN()   (RCC->AHB1ENR |= RCC_AHB1ENR_GPIOAEN)
+/* DAC 0 channel config */
+#define DAC_0_CH0_PIN        4
+#define DAC_0_CH1_PIN        5
+
+/**
+ * @name PWM configuration
+ * @{
+ */
+#define PWM_NUMOF           (2U)
```

Comment:
did you implement a PWM driver? If yes -> add to feature provided in the Makefile, if not -> remove all PWM defines. Same goes for SPI, DAC and I2C below and above.